resources/python/.gitlab-ci.yml

```diff
@@ -0,0 +1,93 @@
+# yamllint disable rule:line-length
+#
+# Copyright (C) 2019 Ultimaker B.V.
+
+include:
+  - project: ultimaker/embedded/prime-jedi
```

Comment:
Maybe specify in the file name that this is specific to embedded projects, since it apparently is?